src/test/ui/async-await/suggest-missing-await.stderr

```diff
@@ -1,6 +1,9 @@
 error[E0308]: mismatched types
   --> $DIR/suggest-missing-await.rs:13:14
    |
+LL | async fn make_u32() -> u32 {
+   |                        --- the `Output` of this `async fn` is the found opaque type
```

Comment:
Still wrong. The opaque type is the `impl Future<...>` and `u32` ends up in `Output = u32` and so therefore the `u32` is "the the `Output` of this `async fn`'s found opaque type".

Reply:
You're absolutely right, rereading this with fresh eyes (don't rewrite diagnostics while tired, people).

I do think that the wording "the Output of this `async fn`'s found opaque type" can be lent to confusion, so it'd be nice to give a specific more verbose version of the output for errors involving `async fn`s, something like:

```
LL | async fn make_u32() -> u32 {
   |                        ---
   |                        |
   |                        this `async fn`'s return type desugars to `impl Future<Output=u32>`
   |                        that return type is the found opaque type
```

That seems like part of a larger effort to make async errors clearer, so I'll go with your proposed copy.